### DISCUSSION TITLE
Connecting GoPro to a cell modem Wi-Fi and controlling it remotely

### DISCUSSION THREAD

**Discussion by Esanii** [gopro/labs#457] [Q&A]
Hello everyone,

Is it possible to connect a GoPro to a cell modem and remotely control the camera?
I want to have a GoPro on a meteorology station, turn the camera on and off remotely, and start and end camera shooting.
I have a **cell modem** and a **CR6 data logger** on the station.

Tnx

**Comment by dnewman-gpsw** [comment]
Not currently possible but on the wish list

**Reply by Esanii** [reply]
Thank you so much for your kind response.

**Comment by compoundvpn** [comment]
Here's an idea that might work for you.  You'll need an Android phone, the AirDrop app, a computer with access to the web version of AirDrop at web.airdrop.com.  Put your cell modem at the remote site with Wifi enabled.  You'll have constant power to the Android phone and your GoPro, and both devices connected to the wifi.  You can remote control into the Android phone, access the Quik app, and remotely start the recording of the camera, and also stop it remotely.  You can enhance security of both the computer and Android by installing the TailScale wireguard app and connect them together.  This way you can monitor your weather station remotely.  If you don't already have wifi remotely but do have the cell modem, just purchase one of theGL.iNet travel routers and plug in the cell modem directly into the USB port of the router.  Hope that helps!
